src/data/int/basic.lean

```diff
@@ -734,6 +742,23 @@ begin
   repeat {assumption}
 end
 
+/-- If an integer with larger absolute value divides an integer, it is
+    zero. -/
```

Comment:
```suggestion
zero. -/
```
I prefer not to indent doc strings like this since sometimes VS Code's markdown renderer gets confused and thinks these are code blocks.